### PR TITLE
Stable v3.3

### PR DESCRIPTION
**Pull request by margau**
- [x] Enable 11s only via multidomain
- [x] Update packages
- [x] Use https://github.com/freifunk-ffm/packages/tree/master/ffffm-autoupdater-use-site-conf-branch to revert 11s-Test back to stable
- [x] Test Build
- [x] Set "GLUON_BRANCH" in site.mk
- [x] Set autoupdater.branch in site.conf
- [x] Set opkg_modules in site.conf
- [x] Update Documentation
- [ ] Merge, Tag & Release

### CODE REVIEW

**Comment by margau** [comment]
11s-test-branch revert moved to #38

File: site.mk [hunk 1]
```diff
@@ -39,18 +38,16 @@ GLUON_MULTIDOMAIN := 1
 # This is the Stable branch
 
 # Gluon Base Release
-DEFAULT_GLUON_RELEASE := v3.2
+DEFAULT_GLUON_RELEASE := v3.3
+
+# Development branch information
+GLUON_BRANCH ?= test
```
Comment (skorpy2009) [inline site.mk]: Ich dachte das wird eine `stable`?

```suggestion
GLUON_BRANCH ?= stable
```

Comment (margau) [reply site.mk]: Ist in der ToDo-Liste ja noch nicht abgehakt ;)

Comment (margau) [reply site.mk]: Fixed in 19d4cd1

**Pull request merged**